src/App.jsx

```diff
@@ -14,57 +16,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodValue, setGoodValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {goodValue ? `${goodValue} is selected` : 'No goods selected'}
```

Comment:
- ternaries are usually formatted the way below, it is much easier to read them this way
- is could be even better readable if we created a variable and moved the result there

```suggestion
        {goodValue 
          ? `${goodValue} is selected` 
           : 'No goods selected'}
```